profile/README.md

```diff
@@ -32,23 +30,21 @@ Our team of dedicated experts is here to help groups and individuals who want to
 
 * [**Hastlayer**](https://hastlayer.com/) turns performance-critical sections of .NET programs into computer chips (FPGA). You can still update your software, but it will be a whole lot faster and more power-efficient.
   
-![Hastlayer_logo](https://user-images.githubusercontent.com/68704777/174298010-59da9056-74f0-48a5-a6f3-ff3f93a2b346.png)
 
 * [**DotNest**](https://dotnest.com/) We've developed DotNest as a software-as-a-service for Orchard CMS, enabling users to create their websites using Orchard in a flash. Today it runs more than 6000 websites.
 
- ![DotNest logo medium](https://user-images.githubusercontent.com/68704777/174301139-3c4ab813-8fe8-4ca1-a428-d9bf4cd98bdb.png)
-
 
 ## Our team
 
 Lombiq's team consists of highly skilled experts from diverse backgrounds:
+
  
- ![lombiq team 2021 summer](https://user-images.githubusercontent.com/68704777/174300662-93d75ff3-5c7d-48c4-9236-775634402287.jpg)
+ <img src="lombiq-team-2021-summer_50.jpg" alt="lombiq team 2021 summer" width="1000"  height="700">
```

Comment:
Why did you change back to an `img` tag when we discussed that you shouldn't use them? Why are there two images in the repo now? We need a single image, which has the correct size and aspect ratio, named lombiq-team-2021-summer.jpg, in Markdown, with an alt text that has correct spelling.

Reply:
Because as I mentioned in the discussion, I cannot add the aspect ratios Zoltan. I tried the 3 following ways mentioned here: 
https://www.youtube.com/watch?v=f4ifdKCH7VI
https://www.youtube.com/watch?v=hHbWF1Bvgf4
 https://www.youtube.com/watch?v=teGy35UGnds
 Same goes for the websites I checked like: 
 https://stackoverflow.com/questions/24383700/resize-image-in-the-wiki-of-github-using-markdown
 and here as well: 
 https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

 The format introduced is the one I used. Other than that, it won't work for me. Please let me know if there is any other source I can work with. And again, thank you for your patience. 
 
 PS: I checked the video tutorial as well, but he didn't have one specific for Picture resizing and aspect ratios. So, I am still checking.
 
![error](https://user-images.githubusercontent.com/68704777/175389078-e432d0a8-b086-42cb-b126-93f73d4f0c7f.PNG)

Reply:
When talking about resizing and aspect ratios, I'm talking about resizing the file, what you already did before. If the file is the correct size, then you don't need to set anything in HTML.

Reply:
I did it last time and you asked why I removed the aspect ratios. That's why I changed it back with the resized file. I misunderstood you then.
So, for now, I must take the team picture (the resized one) and just copy paste it from the repo to the Markdown file. No height width aspects are needed. Is that it?

Reply:
What I said is that images were distorted, because you did resizing without maintaining the aspect ratios. The resizing itself should maintain aspect ratios, not client-side resizing in HTML.

You need to put an appropriately resized file into the repo, as you already did, and reference it from Mardown with the Markdown, not HTML code, what you also did before.

Reply:
I think I understand you better now. Thank you for clarifying your points. I'll submit an updated version right away. I will use the resized image from the repo. I hope this time I get it correct fully :)